MantidPlot/make_package.rb.in

```diff
@@ -314,14 +314,24 @@ end
 `install_name_tool -change #{QtLinkingDir[0]}/QtCore.framework/Versions/4/QtCore @loader_path/../../Frameworks/QtCore.framework/Versions/4/QtCore Contents/MacOS/PyQt4/QtXml.so`
 
 
-#Copy over python libraries not included with OSX. 
+#Copy over python libraries not included with OSX.
 #currently missing epics
 path = "/Library/Python/2.7/site-packages"
 directories = ["sphinx","sphinx_bootstrap_theme","IPython","zmq","pygments","backports","certifi","tornado","markupsafe","jinja2","psutil","jsonschema","functools32","ptyprocess","CifFile","yaml"]
 directories.each do |directory|
   addPythonLibrary("#{path}/#{directory}","Contents/MacOS/")
 end
 
+# System mpl_toolkits in macOS 10.12 and above do not have __init__.py which causes a problem importing
+# So we pack the mpl_toolkits and touch an empty file to temporarily work around it
+#TODO: consider installing mpl_toolkits (thus matplotlib) with pip
+system_python_extras = "/System/Library/Frameworks/Python.framework/Versions/2.7/Extras/lib/python"
+modules = ["mpl_toolkits"]
+modules.each do |directory|
+  addPythonLibrary("#{system_python_extras}/#{directory}","Contents/MacOS/")
+end
+`touch Contents/MacOS/mpl_toolkits/__init__.py`
```

Comment:
This does fix the error, and without it mpl_toolkits will not be a package. It's a shame that full system paths have to be used as will probably make things fragile in the future, but I can't suggest a better way of doing this for now.